### DISCUSSION TITLE
Export table data

### DISCUSSION THREAD

**Discussion by BadaHanDL** [cloudscape-design/components#1665] [Q&A] [answered]
Hi,

How to export table data into Excel or CSV format? Do we have a built-in method to export the data?

Thanks,
Venu

**Comment by johannes-weber** [comment, marked as answer]
There is no build-in method for exporting table data and have no short-term plans to introduce such a feature.
I recommend building your own mechanism to archive the export functionality. 

Thank you for reaching out,
Johannes

**Reply by just-boris** [reply]
Check out this codesadbox with a demo how to implement this feature: https://codesandbox.io/s/cloudscape-export-to-csv-467tr8
